frontend/src/main/java/olumofe/raphael/frontend/Application.java

```diff
@@ -1,7 +1,8 @@
-package starter.project.frontend;
+package olumofe.raphael.frontend;
```

Comment:
package name has changed here. Could you change that across the project? Or revert it here

Reply:
resolved